pext/qml/InstallModuleFromURLDialog.qml

```diff
@@ -65,6 +65,9 @@ Dialog {
 
         xmlhttp.open("GET", textfield.text, true);
         xmlhttp.send();
+
+        destroy();
```

Comment:
Please put this destroy inside the onreadystatechange after installRequest is called

Reply:
Commited! :D